codecov.yml

```diff
@@ -0,0 +1,17 @@
+codecov:
+  branch: master
+coverage:
```

Comment:
Jest has coverage out of the box with `--coverage` so I'm not sure we even need this. See how I intergrated codecov in react-dropzone: https://github.com/okonet/react-dropzone/pull/302/files